README.md

```diff
@@ -1,2 +1,29 @@
-# docs
-Specs and design documentation
+# Nimbess - Specs
+Nimbess is a distributed controller architecture designed to unify network
```

Comment:
what does  unify network policy really mean? What is being unified?

Reply:
unifying network control into a single policy CRD, rather than specifying it in different resources throughout k8s